### ISSUE TITLE
Failing the decloak action freezes the game

### ISSUE THREAD

**Issue by VexingVision** [Sandrem/FlyCasual#2853]
Cloaking Device installed on Zuvio. Cloaking device fails on the planning phase roll. I deliberately fail the decloak by bumping into another ship with a Straight 2.

The game freezes and takes no more inputs.

**Comment by handking88** [comment]
I just experienced the same issue with the Cloaking Device on Drea. Rolling an eye during the start of the Planning Phase, opting to decloak and then deliberately failing the decloak positioning (also used straight 2) should keep the cloak token and allow the Planning Phase to continue, but instead nothing can be set, with no way to proceed.

**Comment by rune-b** [comment]
possibly related to #2859

**Comment by otuv** [comment]
Just had the same issue with Cloaking Device (failing) onto a rock.

**Comment by gausebeck** [comment]
Yup, I've had this same issue with failed decloaks from Cloaking Device.
